src/mono/mono/mini/mini-llvm-cpp.h

```diff
@@ -16,22 +16,14 @@
 #include "llvm-c/Core.h"
 #include "llvm-c/ExecutionEngine.h"
 
+#include "mini-llvm.h"
+
```

Comment:
We didn't include mono headers into this file because it includes mono's config.h etc. which used to confuse the build of mini-llvm-cpp.cpp.

Reply:
Ah. I might just add another file that only contains the enum definition for IntrinsicId then. Or maybe there's an existing header that'd be a more appropriate place to put this (I haven't looked very hard).

Reply:
That would be the best approach imho, we already have a bunch of headers with enums like patch-info.h, wrapper-types.h etc.